### DISCUSSION TITLE
Graphviz with Latex Labels

### DISCUSSION THREAD

**Discussion by julianofoleiss** [quarto-dev/quarto-cli#2633] [Q&A]
Hi everyone!

Is it possible to use latex math as labels in graphviz diagrams with quarto? Specifically, I've been using quarto for RevealJS presentations, and it has been awesome.

Thanks!

**Comment by cscheid** [comment]
I think that's unlikely to work with our out-of-the-box graphviz diagrams, because we're using a webassembly version of graphviz that I'm almost certain doesn't include a latex installation.

On the other hand, we have ObservableJS support, and they _also_ support Graphviz diagrams, specifically SVG ones. I wonder if there's a way to convince MathJax to look at the insides of the diagrams emitted by ObservableJS's `dot` syntax. You'll need some javascript knowledge to do that, but it might actually work.
